packages/rn-tester/js/examples/TurboModule/NativeCxxModuleExampleExample.js

```diff
@@ -261,7 +261,7 @@ class NativeCxxModuleExampleExample extends React.Component<{||}, State> {
   }
 
   componentDidMount(): void {
-    if (global.__turboModuleProxy == null) {
+    if (global.nativeModuleProxy == null) {
```

Comment:
@zhongwuzw, this is always false! (in bridge and bridgeless mode, nativeModuleProxy is always non-null)

What do you think we should do here?

Reply:
@RSNara Sorry, my bad, maybe I missed something when I test the issue. :) I updated the commit.